quality/findbugs/android.gradle

```diff
@@ -5,20 +5,23 @@ findbugs {
   effort = 'max' //  min, default, or max.
   ignoreFailures = false
   excludeFilter = rootProject.file('config/quality/findbugs/findbugs-filter.xml')
-  reportLevel = 'low' // low, medium, high 
+  reportLevel = 'low' // low, medium, high
 }
 
 check.dependsOn 'findbugs'
 task findbugs(type: FindBugs,
     group: 'verification',
     description: 'findbugs checks for main and test java sourcesets of android projects',
-    dependsOn: ['compileDebugSources', 'compileReleaseSources']) {
+    dependsOn: [ 'compileDebugSources', 'compileReleaseSources', 'assemble' ]) {
```

Comment:
From the code below it seems you only need to depend on `assembleRelease`

Reply:
Also, I guess the `compile…Sources` tasks don't need to be specified now

Reply:
removed those